dmoj/checkers/floats.py

```diff
@@ -27,7 +28,13 @@ def verify_default(process_float, judge_float, epsilon):
             abs(1.0 - process_float / judge_float) <= epsilon)
 
 
-def check(process_output, judge_output, precision=6, error_mode='default', **kwargs):
+def check(process_output, judge_output, precision=6, fast=True, error_mode='default', **kwargs):
+
+    if fast:
```

Comment:
Having people write `fast: False` is kind of a weird way to say that they want arbitrary-precision calculations...